tests/cases/fourslash/completionConstructorKeywordAfterPropertyDeclaration.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference path="fourslash.ts" />
+
+////class A {
+////  bla
+////  constructor/*1*/(props) {}
```

Comment:
The presence of the full `constructor` keyword is changing the behavior of the completions. It seems like what you really want to test is

```ts
class A {
  bla
  /**/
}
```

and

```ts
class A {
  bla
  co/**/
}
```

right? Those currently _don’t_ work.

Reply:
We’ll have to revisit my comment about newlines after this issue is fixed—you’re right that the behavior is currently unchanged, but I think it’s because of this issue.

Reply:
Got it. This is a more universal situation. And I've found that completions are not working either for snippet like this:

```ts
class C {
  blah = 123 // property with an immediate assignment
  constructor/**/
}
```

I'll take a deeper dive into these problems.